lib/omnibus/health_check.rb

```diff
@@ -124,6 +124,12 @@ class HealthCheck
       /libgcc_s.so/,
       /libstdc\+\+\.so/,
       /libcrypt.so/,
+      /libcrypto.so/,
+      /libssl.so/,
```

Comment:
These are a bit worrying.  Shouldn't these always be safely provided by OpenSSL?